extensions/MozChangeField/lib/Post/CommentOnSeverity.pm

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::MozChangeField::Post::CommentOnSeverity;
+
+use 5.10.1;
+use Moo;
+
+use Bugzilla::Error;
+use Bugzilla::Util qw(trim);
+
+use constant DEFAULT_COMMENT =>
+  'Changing severity to S? because of <rationale>.';
+
+sub evaluate_create {
```

Comment:
A comment is only required when _changing_ the severity field, not during bug creation.

Remove all code relating to bug creation.